Pod/Classes/TemporaryAPI/ARTemporaryAPIModule.m

```diff
@@ -9,9 +9,19 @@ @implementation ARTemporaryAPIModule
   self.artistFollowStatusProvider(artistID, block);
 }
 
-RCT_EXPORT_METHOD(setFollowStatus:(BOOL)status artistID:(NSString *)artistID completion:(RCTResponseSenderBlock)block)
+RCT_EXPORT_METHOD(setFollowArtistStatus:(BOOL)status artistID:(NSString *)artistID completion:(RCTResponseSenderBlock)block)
```

Comment:
_note this_ - the API inside JS would be `setFollowStatus` but you can't have two of them, so I gave them unique names